great_expectations/dataset/sqlalchemy_dataset.py

```diff
@@ -375,7 +374,7 @@ def _get_count_query_mssql(
         ignore_values_condition: BinaryExpression,
     ) -> Select:
         # mssql expects all temporary table names to have a prefix '#'
```

Comment:
Is this comment still needed? Does this function handle it based on dialect?

Reply:
No -- deleted.  Great catch again (deleted from V3, but missed V2) -- please verify!  Thanks!